bcda/cclf/cclf.go

```diff
@@ -255,7 +255,7 @@ func importCCLF(fileMetadata *cclfFileMetadata, importFunc func(uint, []byte, *g
 func getCCLFArchiveMetadata(filePath string) (cclfFileMetadata, error) {
 	var metadata cclfFileMetadata
 	// CCLF filename convention for SSP with BCD identifier: P.BCD.ACO.ZC0Yyy.Dyymmdd.Thhmmsst (timestamp will include the ACO ID value)
-	filenameRegexp := regexp.MustCompile(`(T|P)\.BCD\.ACO\.ZC(0|8)Y(\d{2})\.(D\d{6})\.T(\d{4})\d{3}`)
+	filenameRegexp := regexp.MustCompile(`(T|P)\.BCD\.ACO.*\.ZC(0|8)Y(\d{2})\.(D\d{6})\.T(\d{4})\d{3}`)
```

Comment:
`ACO.*` is quite wide open.
Should we update to `ACOB?` so that it matches either `ACO` or `ACOB`?

Reply:
Yeah I totally agree, I ended up reverting it back to what we had before though. We could support both just (ACO and ACOB) with something like 'ACO(|B)' but then the indexes would be different. I can do that no problem, just didn't wanna modify too much. What u think??

Reply:
If you do it without a group then it will probably not affect your indexes... try `ACOB?`

Reply:
just pushed up, good stuff